translations/default/ModulesGoogleanalyticsAdmin.xlf

```diff
@@ -0,0 +1,185 @@
+        <note>Line: 152</note>
+      </trans-unit>
+      <trans-unit id="d5448915ef8577d6b08407c0b4136afb">
+        <source>Re-send failed orders</source>
+        <target>Re-send failed orders</target>
+        <note>Line: 163</note>
+      </trans-unit>
+      <trans-unit id="4c3222a797a0f86905fab4e092c1cfc9">
+        <source>This option will resend all orders that failed to be sent normally in front-office, due to failures or ad-blockers.</source>
+        <target>This option will resend all orders that failed to be sent normally in front-office, due to failures or ad-blockers.</target>
+        <note>Line: 165</note>
+      </trans-unit>
+      <trans-unit id="7e3e525741740ee3418936b003eabbca">
+        <source>Failed orders period</source>
+        <target>Failed orders period</target>
+        <note>Line: 181</note>
+      </trans-unit>
+      <trans-unit id="205277b9a12a73980e07b1ac1add0a7b">
+        <source>If you want to resend failed orders, specify how many days back the module should look for them. Default: 30.</source>
+        <target>If you want to resend failed orders, specify how many days back the module should look for them. Default: 30.</target>
+        <note>Line: 185</note>
+      </trans-unit>
+      <trans-unit id="8dd2f915acf4ec98006d11c9a4b0945b">
+        <source>Settings updated successfully.</source>
+        <target>Settings updated successfully.</target>
+        <note>Line: 251</note>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="modules/ps_googleanalytics/ps_googleanalytics.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="d86cf69a8b82547a94ca3f6a307cf9a6">
+        <source>Google Analytics</source>
+        <target>Google Analytics</target>
+        <note>Line: 58</note>
+      </trans-unit>
+      <trans-unit id="135d2522825fa02688ab25a2e1c88c73">
+        <source>Gain clear insights into important metrics about your customers, using Google Analytics</source>
+        <target>Gain clear insights into important metrics about your customers, using Google Analytics</target>
+        <note>Line: 59</note>
+      </trans-unit>
+      <trans-unit id="7ed5c154078e30b30b2f214299c5e9f5">
+        <source>Are you sure you want to uninstall Google Analytics? You will lose all the data related to this module.</source>
+        <target>Are you sure you want to uninstall Google Analytics? You will lose all the data related to this module.</target>
+        <note>Line: 60</note>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="modules/ps_googleanalytics/views/templates/admin/configuration.tpl" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="deb905422d07a5bb74cb3178df580e81">
+        <source>Your customers go everywhere; shouldn't your analytics.</source>
+        <target>Your customers go everywhere; shouldn't your analytics.</target>
```

Comment:
What does it mean? Is it a question?

Reply:
This string was there for years, some promo sentence.

Reply:
Ok, but it doesn't mean anything. 😅  Since I've seen it, maybe we could take the opportunity to change it? 👀

Reply:
@Julievrz Can't anything other than agree. 🤣 We can just remove that whole block from the module, because it's outdated anyway. And replace it with some textual info what that module does, what events it can send etc.